platform.mk

```diff
@@ -94,6 +94,12 @@ TARGET_NEEDS_ADSP_SENSORS_PDR := true
 # VPP
 TARGET_DISABLE_QTI_VPP := true
 
+# Hvdcp_opti
+TARGET_NEEDS_HVDCP_OPTI := true
+
+# Cdspstart
+TARGET_NEEDS_CDSPSTART := true
```

Comment:
I feel like this should go under `# DSP` just a few lines above. Similarly, on second thought, I don't `TARGET_NEEDS_CDSPSTART` is a reusable name. Instead, if we have `TARGET_HAS_CDSP` we can later reuse that if we have more CDSP-specific features.

What do you think?

Reply:
Sure thing. Give me a sec. Have some repos to rebase :)